.travis.yml

```diff
@@ -7,10 +7,15 @@ php:
   - 5.6
 
 env:
-  - SYMFONY_VERSION="2.3.*"
-  - SYMFONY_VERSION="2.5.*"
   - SYMFONY_VERSION="2.6.*"
 
+matrix:
+  include:
+    - php: 5.6
+      SYMFONY_VERSION="2.5.*"
```

Comment:
I would recommend to remove 2.5 tests, as the version reached end of maintance in January.